packages/db/test/utils/index.coffee

```diff
@@ -1,18 +1,101 @@
-{db} = require '../../src/utils'
-
-describe 'utils', ->
+utils = require '../../src/utils'
+{command, escape, jdbc} = utils.db
 
+describe "db.utils", ->
+  
+  describe "escape", ->
+  
+    it 'backslashes', ->
+      escape('\\').should.eql '\\\\'
+  
+    it 'double quotes', ->
+      escape('"').should.eql '\\"'
+  
+    it 'backslashes and double quotes', ->
+      query = 'SELECT * FROM my_db WHERE name = "John\\\'s"'
+      expected = 'SELECT * FROM my_db WHERE name = \\"John\\\\\'s\\"'
+      escape(query).should.eql expected
+      
   describe 'jdbc', ->
 
       it 'get default port', ->
-        db.jdbc('jdbc:mysql://localhost/my_db').should.eql
+        jdbc('jdbc:mysql://localhost/my_db').should.eql
           engine: 'mysql',
           addresses: [ { host: 'localhost', port: 3306 } ],
           database: 'my_db'
-        db.jdbc('jdbc:postgresql://localhost/my_db').should.eql
+        jdbc('jdbc:postgresql://localhost/my_db').should.eql
           engine: 'postgresql',
           addresses: [ { host: 'localhost', port: 5432 } ],
           database: 'my_db'
 
       it 'get database', ->
-        db.jdbc('jdbc:mysql://master3.ryba:3306/my_db?a_param=true').database.should.eql 'my_db'
+        jdbc('jdbc:mysql://master3.ryba:3306/my_db?a_param=true').database.should.eql 'my_db'
+          
+  describe 'command', ->
+    
+    it 'invalid engine', ->
+      () -> command
+        engine: 'invalid_engine'
+      .should.throw 'Unsupported engine: "invalid_engine"'
+      
+    describe "using engine: mariadb", ->
+      
+      it "default arguments", ->
+        command
+          admin_password: 'rootme'
+          admin_username: 'root'
+          engine: 'mariadb'
+          host: 'localhost'
+          port: 3306
+        .should.equal 'mysql -hlocalhost -P3306 -uroot -p\'rootme\''
+      
+      it 'set default port engine', ->
+        command
+          admin_password: 'rootme'
+          admin_username: 'root'
+          engine: 'mariadb'
+          host: 'localhost'
+        .should.equal 'mysql -hlocalhost -P3306 -uroot -p\'rootme\''
+    
+      it 'command option', ->
+        command
+          admin_password: 'rootme'
+          admin_username: 'root'
+          engine: 'mariadb'
+          host: 'localhost'
+          port: 3306
```

Comment:
If you test user options, then it makes sense to use different values than the default ones.